pac4j-cas/src/main/java/org/pac4j/cas/credentials/authenticator/CasRestAuthenticator.java

```diff
@@ -49,8 +55,13 @@ public void validate(final UsernamePasswordCredentials credentials, final WebCon
             throw new TechnicalException("Credentials are required");
         }
         final String ticketGrantingTicketId = requestTicketGrantingTicket(credentials.getUsername(), credentials.getPassword(), context);
-        final CasRestProfile profile = new CasRestProfile(ticketGrantingTicketId, credentials.getUsername());
-        credentials.setUserProfile(profile);
+		if (Strings.isNullOrEmpty(ticketGrantingTicketId)) {
```

Comment:
org.pac4j.core.exception.TechnicalException: TGT cannot be blank
	at org.pac4j.core.util.CommonHelper.assertTrue(CommonHelper.java:134)
	at org.pac4j.core.util.CommonHelper.assertNotBlank(CommonHelper.java:145)

CommonHelper.assertNotBlank("TGT", ticketGrantingTicketId);
@leleuj: I added this line before set profile. But I think it's not correct. We still receive TechnicalException for CAS failed authentication.
Is it ok?